site/en/docs/react_gemini_prompting.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        """perfoms search on <code>query</code> via wikipedia api and returns it's summary.
</pre><p> </p><p><code>its</code> not <code>it's</code> (and as above, use inital-caps here)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>